scripts/make_readme_pdf.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# exit on error
+set -e
+
+# directory of this script
+DIR=$(realpath $(dirname $0))
+
+pushd $DIR/..
+pandoc README.md -t html5 -o README.pdf
```

Comment:
This requires installation of pandoc to compile the PDF. That should be documented somewhere or entered in a requirements_docu.txt. Maybe time to make a `CONTRIBUTING.md`?

Reply:
Ha, never created one of those. But yeah for sure, those requirements are needed, forgot to add them.